index.html

```diff
@@ -1028,6 +1061,461 @@ <h2>
+            will resolve to when the operation does not complete because the
+            {{Document}} is `"hidden"` or another effect was started
+            on the same actuator.
+          </p>
+        </dd>
+      </dl>
+    </section>
+    <section>
+      <h3>
+        <dfn>GamepadHapticEffectType</dfn> Enum
+      </h3>
+      <p>
+        The effect type defines how the effect parameters are interpreted by
+        the actuator.
+      </p>
+      <p>
+        The [=user agent=] MAY modify the behavior of
+        {{GamepadHapticActuator}}.{{GamepadHapticActuator/playEffect}} to
+        extend support for new {{GamepadHapticEffectType}} values and
+        {{GamepadEffectParameter}} attributes. Implementations MUST NOT
+        modify the definition of a [=valid effect=] for effect types defined
+        in this specification.
+      </p>
+      <pre class="idl">
+        enum GamepadHapticEffectType {
+          "dual-rumble"
+        };
+      </pre>
+      <dl>
+        <dt>
+          <dfn data-for="GamepadHapticEffectType">"dual-rumble" effect
+          type</dfn>
+        </dt>
+        <dd>
+          <p>
+            {{GamepadHapticEffectType/"dual-rumble"}} describes a haptics
+            configuration with an eccentric rotating mass (ERM) vibration
+            motor in each handle of a standard gamepad. In this
+            configuration, either motor is capable of vibrating the whole
+            gamepad. The vibration effects created by each motor are unequal
+            so that the effects of each can be combined to create more
+            complex haptic effects.
+          </p>
+          <p>
+            A {{GamepadHapticEffectType/"dual-rumble"}} effect is a
+            fixed-length, constant-intensity vibration effect intended for an
+            actuator of this type. {{GamepadHapticEffectType/"dual-rumble"}}
+            effects are defined by {{GamepadEffectParameters/startDelay}},
+            {{GamepadEffectParameters/duration}},
+            {{GamepadEffectParameters/strongMagnitude}}, and
+            {{GamepadEffectParameters/weakMagnitude}}, none of which are
+            required.
+          </p>
+          <p>
+            {{GamepadEffectParameters/startDelay}} and
+            {{GamepadEffectParameters/duration}} set the duration of the
+            initial delay before after initiating the effect before the
+            effect will be started
+          </p>
+          <p>
+            {{GamepadEffectParameters/startDelay}} sets the duration of the
+            delay after {{GamepadHapticActuator/playEffect()}} is called
+            until vibration is started, in milliseconds. During the delay
+            interval, the actuator must not vibrate. An effect with negative
+            {{GamepadEffectParameters/startDelay}} is not a [=valid effect=].
+            If the {{GamepadEffectParameters/startDelay}} is not specified,
+            the default delay is 0 milliseconds.
```

Comment:
Why is startDelay a `double` when it doesn't allow negative and it represents milliseconds anyway? Should this just just a `unsigned long long` instead?